lib/jsonapi-serializers/serializer.rb

```diff
@@ -286,7 +285,6 @@ def self.serialize(objects, options = {})
         'data' => primary_data,
       }
       result['meta'] = options[:meta] if options[:meta]
-      result['errors'] = options[:errors] if options[:errors]
```

Comment:
Yeah, you're totally right! My bad. I've rolled back the support for `:errors` option and maintained the existing spec so we know if something breaks even with the deprecated option.